doc/news/changes/minor/20190411TimoHeister

```diff
@@ -0,0 +1,3 @@
+If present, we will detect the LLD linker ld.lld at configuration time.
```

Comment:
~~ld.lld~~ lld  (?)

Reply:
nevermind, I see that it's use this way above.

Reply:
Yeah, the binary is typically called ld.lld on your system.